src/widgets/configure/__tests__/configure-test.js

```diff
@@ -0,0 +1,85 @@
+import { SearchParameters } from 'algoliasearch-helper';
+import configure from '../configure';
+
+describe('configure', () => {
+  it('throws when you pass it a non-plain object', () => {
+    [
+      () => configure(new Date()),
+      () => configure(() => {}),
+      () => configure(/ok/),
+    ].map(widget => expect(widget).toThrowError(/Usage/));
```

Comment:
Why a map?

Reply:
no reason, can be `forEach` too

Reply:
We we perform side effects it's better to use a `forEach`. With it the intent is clear: I will perform a side effect. We should not use `map` *only* for iterate on an array. The main purpose of a `map` is to transform an array of `A` into an array of `B`.